yt/docker/ya-build/query-tracker/build_yql_agent.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+# This script is expected to be executed from `build_query_tracker_in_docker.sh` script. See it for more details.
+
+set -eux
+shopt -s expand_aliases
+
+export YTSAURUS_SOURCE_PATH=$(realpath $YTSAURUS_SOURCE_PATH)
+export YDB_SOURCE_PATH=$(realpath $YDB_SOURCE_PATH)
+export YQL_BUILD_PATH=$(realpath $YQL_BUILD_PATH)
+
+mkdir -p $YTSAURUS_BUILD_PATH
+mkdir -p $YQL_BUILD_PATH
+
+apt-get update
+DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install -y software-properties-common build-essential
+DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC add-apt-repository -y ppa:deadsnakes/ppa
+apt-get update
+DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install -y python3.8-dev python3.9-dev python3.10-dev python3.11-dev python3.12-dev
+
+# Build yql agent binary
+${YTSAURUS_SOURCE_PATH}/ya make -T ${BUILD_FLAGS} --output=${YQL_BUILD_PATH} ${YTSAURUS_SOURCE_PATH}/yt/yql/agent/bin
+
+# Build required binaries and libraries
+for path in "ydb/library/yql/tools/mrjob" \
+            "ydb/library/yql/yt/dynamic" \
+            "ydb/library/yql/yt/dq_vanilla_job" \
+            "ydb/library/yql/yt/dq_vanilla_job.lite" \
+            "ydb/library/yql/udfs/logs/dsv"
+do
+    ${YDB_SOURCE_PATH}/ya make -T ${BUILD_FLAGS} --ignore-recurses --output=${YQL_BUILD_PATH} ${YDB_SOURCE_PATH}/$path
+done
+
+# Build common yql udfs
+for udf_name in compress_base \
+                datetime2 \
+                digest \
+                file \
+                histogram \
+                hyperloglog \
+                hyperscan \
+                ip_base \
+                json \
+                json2 \
+                math \
+                pire \
+                protobuf \
+                re2 \
+                set \
+                stat \
+                streaming \
+                string \
+                top \
+                topfreq \
+                unicode_base \
+                url_base \
+                yson2
+do
+    ${YDB_SOURCE_PATH}/ya make -T ${BUILD_FLAGS} --ignore-recurses -DSTRIP=yes --output=${YQL_BUILD_PATH} ${YDB_SOURCE_PATH}/ydb/library/yql/udfs/common/${udf_name}
```

Comment:
Why are we starting to ignore STRIP flag here?

Reply:
I looked at python libraries and saw that they are always stripped:
https://github.com/ytsaurus/ytsaurus/blob/b47834934522a0d9d07f98c087fa8aaf3b56e63b/.github/workflows/build-jobs.yaml#L232

I decided that it makes sense to strip these .so's as well. Isn't this the case?